brave/src/test/java/brave/secondary_sampling/integration/SecondarySamplingIntegratedTest.java

```diff
@@ -375,4 +375,31 @@
   @After public void close() {
     Tracing.current().close();
   }
+
+  static int traceCount(TestSpanHandler spans) {
+    return (int) spans.spans().stream().map(MutableSpan::traceId).distinct().count();
+  }
+
+  static List<DependencyLink> getDependencies(TestSpanHandler spans) {
```

Comment:
it is possible we could port the dependency linker to brave-tests.. just now don't feel like it.